contexts/credentials/v2

```diff
@@ -1,6 +1,7 @@
 {
   "@context": {
     "@protected": true,
+    "@vocab": "https://www.w3.org/ns/credentials/issuer-dependent#",
```

Comment:
This works for me.

Reply:
@iherman yes... I discussed this with @msporny... I had same thought... This was the proposal that came from that... If this discussion balloons because of that choice, I suggest we revert to 2018.